### ISSUE TITLE
Tapping the reply action button no longer focuses on the reply/thread sidebar text field

### ISSUE THREAD

**Issue by stakx** [mattermost/mattermost#28292]
#### Summary

Unless I am mistaken, clicking the _Reply_ action button on a post previously opened the RHS reply/thread sidebar and moved the input focus to the text field at the bottom. This behavior has apparently changed with Mattermost v10, the input focus is no longer set.

#### Steps to reproduce

![image](https://github.com/user-attachments/assets/48bef945-7b2d-4c10-be51-6f1615f8e80a)

1. Click on the _Reply_ action button on any post (indicated with an arrow in the screenshot above).
2. Take note of which text field received the input focus (if any).

#### Operating System

Windows 11 Pro (23H2, Build 22631.4169)

#### Web browser used

Firefox 130.0.1 (64-bit) with all add-ons / extensions disabled

#### Mattermost Server Version

10.0.0 (Build 10671348655)

#### Expected behavior

I expected the RHS reply/thread sidebar to open & for the input focus to shift to the text field there (marked as 2 in the screenshot above).

#### Observed behavior

The sidebar opens but the input focus is not always set to its text field. (It works some of the time, but not always.)

#### Additional Information

* If I remember correctly, this new, somewhat unintuitive behavior started with Mattermost v10 but worked fine before that.
* I posted an additional [screencast](https://github.com/user-attachments/assets/5a8829db-1235-4746-8936-c523caad4401) in [my next post below](https://github.com/mattermost/mattermost/issues/28292#issuecomment-2385536529).

**Comment by stakx** [comment]
@amyblais, yes, the same thing happens on https://community.mattermost.com/. I've recorded a brief [screencast](https://github.com/user-attachments/assets/5a8829db-1235-4746-8936-c523caad4401) demonstrating how clicking on a post's _Reply_ action icon loses the input focus.

I've since observed two different behaviors: sometimes after clicking on _Reply_ (1) the input focus moves to the text field that had the focus last (even if it's the wrong one), and sometimes (2) the input focus is lost completely (as shown in the screencast).

(Btw. I disabled all browser add-ons / extensions for my testing, so there's no interference from those.)
